ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -670,6 +670,10 @@ then
 	then
 		mavlink start -d ${MAVLINK_COMPANION_DEVICE} -b 115200 -m minimal -r 1000
 	fi
+	if param compare SYS_COMPANION 6460800
```

Comment:
Any reason to establish the RTPS client to half the baud rate of the ESP8266?

Reply:
I was following another example at the time using this baudrate. Feel free to change it to a more appropriate default, or add a range of desired rates.